### ISSUE TITLE
Compress display of pipes on the sidebar

### ISSUE THREAD

**Issue by anewuser** [pipes-digital/pipes#21]
With the current setup, the more pipes you create, the more you'll have to scroll down to reach the save button, especially if they have long titles. You could add the existing pipes to a unified field with a dropdown menu for the user to choose from, or at least move the save button to a fixed position at the top of the window as a temporary workaround.

![Sidebar](https://user-images.githubusercontent.com/6733770/28387143-50c68cbc-6ca4-11e7-8c47-91fd3771caae.png)

**Comment by onli** [comment]
You probably saw that, the block buttons are not behind a menu. Does that work better for you?

**Comment by anewuser** [comment]
The problem continues because the current pipes are now listed along with the input blocks (which will _always_ be used and activated for creating a new pipe, right?). The first thing I did when I started creating a new pipe was to expand the "Inputs" section, which thanks to the long titles of my two other pipes pulled everything else out of the viewport (I have a 1366x768 monitor), so I had to scroll down even more now to see the other blocks (it's either that or having to keep collapsing/expanding the sections). 

My original suggestion was collapse only the other pipes and keep all the other site features/blocks visible by default, so that I and other users with similar resolutions/long pipe titles would be able to see everything (including the save button) without scrolling down or having to use extra clicks to show/hide stuff.

![Viewport](https://user-images.githubusercontent.com/6733770/29131375-0efc8cd6-7d03-11e7-978c-220e08c76e30.png)
